media/css/pebbles/components/_buttons-download.scss

```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 @import '../includes/lib';
+@import '../includes/fonts/font-awesome';
```

Comment:
It would be nice to avoid needing to load font-awesome just for one tiny icon. I'd suggest using a simple png instead here.

Reply:
Thanks @alexgibson! Will replace Font Awesome with a small SVG file.

Reply:
I think SVG is likely not to be shown on older systems, so I'd either find a png or drop the icon.